### PR TITLE
Dtspo 14094

### PR DESCRIPTION
**Pull request by SalimKainos**
Please remove this line and everything above and fill the following sections:
https://github.com/hmcts/pre-shared-infrastructure/pull/518
https://github.com/hmcts/martha-infra/pull/62

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DTSPO-14095
https://tools.hmcts.net/jira/browse/DTSPO-14094

### Change description ###

Fixes https://github.com/hmcts/terraform-module-virtual-machine/issues/19
Fixes https://github.com/hmcts/terraform-module-virtual-machine/issues/24

Fix Availability Zones variable - Typo
Privateip-allocation variable default to dynamic

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: example/main.tf [hunk 1]
```diff
@@ -11,7 +11,7 @@ module "virtual-machine" {
   vm_sku               = "22.04-LTS"
   vm_size              = "D2ds_v5"
   vm_version           = "latest"
-  vm_availabilty_zones = "1"
+  vm_availability_zones = "1"
```
Comment (timja) [inline example/main.tf:14]: this file needs formatting

**Review by timja — CHANGES_REQUESTED** [review]
Are you sending pull requests to all consumers to fix the typo? what is the migration plan?

**Comment by SalimKainos** [comment]
> Are you sending pull requests to all consumers to fix the typo? what is the migration plan?

Yep.Two repos using this module have same typo thats y i'm sending pull requests.

**Review by timja — APPROVED** [review]
> > Are you sending pull requests to all consumers to fix the typo? what is the migration plan?
> > Yes, that's correct. Two repositories utilizing this module have the same typo, which is why I'm submitting pull requests to address it.

I see more than 2 results here:
https://github.com/search?q=org%3Ahmcts%20vm_availabilty_zones&type=code

can you link the PRs please?

**Pull request merged**